pages/applications/edit/[id]/[person]/index.tsx

```diff
@@ -94,12 +98,12 @@ export default function EditApplicant({ user, data }: PageProps): JSX.Element {
 
     setIsSubmitted(true);
   };
-
+  /* eslint-disable react/jsx-no-useless-fragment */
   return (
     <>
```

Comment:
Really curious how on some of these fragments, if you remove the empty div it has an issue with the data. Can't quite get my head around why expect maybe the linter is reading it wrong.

Reply:
I just disabled that for now, but definitely worth investigating further at later date